features/step_definitions/helper_services.rb

```diff
@@ -10,7 +10,7 @@
 
   step %Q/SCC "privileged" is added to the "system:serviceaccounts:<%= project.name %>" group/
   step %Q{I run the :create client command with:}, table(%{
-    | f | #{ENV['BUSHSLICER_HOME']}/testdata/storage/nfs/nfs-server.yaml |
+    | f | <%= BushSlicer::HOME %>/testdata/storage/nfs/nfs-server.yaml |
```

Comment:
@liangxia , actually to make sure that whenever we decide to keep test data, we can quickly update all places, even if it is in a separate repo again, it is much better to do like 
```
  step 'I obtain test data file "testdata/storage/nfs/nfs-server.yaml"'
  step %Q{I run the :create client command with:}, table(%{
    | f | <%= BushSlicer::HOME %>/testdata/storage/nfs/nfs-server.yaml |
  })